{"name": "test_successful_login", "status": "broken", "statusDetails": {"message": "AttributeError: module 'urllib.request' has no attribute 'config'", "trace": "@pytest.fixture(scope=\"function\", autouse=True)\n    def setup_browser():\n>       browser_name = request.config.getoption('browser_name')\nE       AttributeError: module 'urllib.request' has no attribute 'config'\n\ntests\\conftest.py:31: AttributeError"}, "start": 1689778385893, "stop": 1689778385893, "uuid": "098e55b2-52c8-45b3-ab2d-a0b0e34a025d", "historyId": "f4e42ed032845b0ed5c71539b726bc6e", "testCaseId": "f4e42ed032845b0ed5c71539b726bc6e", "fullName": "tests.test_login#test_successful_login", "labels": [{"name": "feature", "value": "Sign in"}, {"name": "tag", "value": "web"}, {"name": "story", "value": "Successful login and expected redirection of URL"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "LAPTOP-OFQSKCIR"}, {"name": "thread", "value": "16652-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}